### PR TITLE
Story Promo Timestamp update - enable dual calendars

### LINKED ISSUE #6081: Update Persian Homepage to use dual calendars
**Is your feature request related to a problem? Please describe.**
The current PAL homepage for Persian shows the date as using a dual calendar format. The date format on the new Paersian homepage is only showing 1 calendar. This should show a dual calendar (as seen in other Simorgh pages). 

The dual calendar format example can be seen here
https://www.bbc.com/persian/world-52126433

**Describe the solution you'd like**
Any date on the Persian homepage should use the dual calendar format

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
Homepage
image

Simorgh page
https://www.bbc.com/persian/world-52126433
image

### PR DESCRIPTION
**Pull request by sareh**
Resolves #6081 #6080

**Overall change:**
Dual calendars for Pashto & Persian are now showing on Story Promos on the front page and on CPS pages that use StoryPromos.

**Code changes:**

- Passed altCalendar to Timestamp within StoryPromo
- Added unit tests for Story Promo time element for Pashto & Persian
- Added STY assets for Pashto & Persian so Story Promo can be viewed locally.

You can see these dual timestamps on all Pashto and Persian pages that have Story Promos.

Story Promo on Persian front page, with dual calendar timestamp http://localhost:7080/persian
Screenshot Persian
Story Promo on Pashto front page, with dual calendar timestamp http://localhost:7080/pashto
Screenshot Pashto

Note that it should also be seen every other place that Story Promos - e.g. STY pages that have CPS Related Content, CPS Top Stories, CPS Features Analysis. 
http://localhost:7080/pashto/columns-52106293
http://localhost:7080/persian/arts-52166891

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by tochwill — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
Looks good 👍

File: src/app/containers/StoryPromo/index.test.jsx [hunk 1]
```diff
@@ -349,6 +349,31 @@ describe('StoryPromo Container', () => {
...
+        expectationFirstJan2020: '۱۱ دی ۱۳۹۸ - ۱ ژانویه ۲۰۲۰',
+      },
+    ].forEach(({ service, expectationFirstJan2020 }) => {
+      it(`should render time element with multiple calendars for ${service} Story Promo`, () => {
+        const firstJan2020 = 1577836800000;
+        const newItem = {
+          ...cpsItem,
+          timestamp: firstJan2020,
+        };
+        const persianContainer = render(
```
Comment (DenisHdz) [inline src/app/containers/StoryPromo/index.test.jsx]:
```suggestion
        const { container } = render(
            <WrappedStoryPromo service={service} item={newItem} />,
         );
```

File: src/app/containers/StoryPromo/index.test.jsx [hunk 2]
```diff
@@ -349,6 +349,31 @@ describe('StoryPromo Container', () => {
...
+          <WrappedStoryPromo service={service} item={newItem} />,
+        ).container;
+        expect(
+          persianContainer.getElementsByTagName('time')[0].innerHTML,
```
Comment (DenisHdz) [inline src/app/containers/StoryPromo/index.test.jsx]:
```suggestion
          container.getElementsByTagName('time')[0].innerHTML,
```

File: src/app/containers/StoryPromo/index.test.jsx [hunk 3]
```diff
@@ -349,6 +349,31 @@ describe('StoryPromo Container', () => {
...
+    ].forEach(({ service, expectationFirstJan2020 }) => {
+      it(`should render time element with multiple calendars for ${service} Story Promo`, () => {
+        const firstJan2020 = 1577836800000;
+        const newItem = {
+          ...cpsItem,
+          timestamp: firstJan2020,
+        };
+        const persianContainer = render(
+          <WrappedStoryPromo service={service} item={newItem} />,
+        ).container;
```
Comment (EinsteinNjoroge) [inline src/app/containers/StoryPromo/index.test.jsx]:
```suggestion
        );
```
Following Denis' suggestion

Comment (sareh) [reply src/app/containers/StoryPromo/index.test.jsx]: Thanks, this change was made.

File: src/app/containers/StoryPromo/index.test.jsx (hunk 1 above)
Comment (sareh) [reply src/app/containers/StoryPromo/index.test.jsx]: Thanks, this change was made.

**Review by DenisHdz — APPROVED** [review]
LGTM

**Pull request merged**